src/Command/RegisterChangedSchemasCommand.php

```diff
@@ -60,20 +70,28 @@ public function execute(InputInterface $input, OutputInterface $output): int
 
         while (false === $this->abortRegister) {
             $failed = [];
```

Comment:
this should be outside of the loop, i want to see all failed together and all succeeded together. you probably need a key (schema name or so), to not have duplicate entries because of retires

Reply:
I can just distinct values. Can be moved outside